torch/csrc/distributed/c10d/TraceUtils.h

```diff
@@ -479,13 +490,15 @@ struct NCCLTraceBuffer {
   c10::optional<size_t> record(
       size_t pg_id,
       const std::tuple<std::string, std::string>& pg_name,
-      size_t seq_id,
+      size_t collective_seq_id,
+      size_t p2p_seq_id,
       size_t op_id,
       std::string profiling_name,
       const std::vector<at::Tensor>& inputs,
       const std::vector<at::Tensor>& outputs,
       Event* start,
-      Event* end) {
+      Event* end,
+      bool isP2P = "false") {
```

Comment:
`bool isP2P = false`, otherwise by default this is setting it to true. Probably better not to use a default argument here and just update the call sites.

Reply:
Good suggestion. Fixed!